app/lib/snippet-view.dart

```diff
@@ -0,0 +1,21 @@
+// import 'package:flutter/material.dart';
+
+// import 'models/snippet.dart';
+
+// class SnippetView extends StatefulWidget {
+//   final Snippet snippet;
+//   SnippetView({Key key, this.snippet}) : super(key: key);
+//   @override
+//   _SnippetViewState createState() => _SnippetViewState(snippet: snippet);
+// }
+
+// class _SnippetViewState extends State<SnippetView> {
+
+//   _SnippetViewState(this.snippet, {Object snippet});
+//   @override
+//   Widget build(BuildContext context) {
+//     return Container(
+      
+//     );
+//   }
+// }
```

Comment:
Why is this commented out?

Reply:
This is not finished yet, and if I don't comment it out it will give everyone build errors

Reply:
ok